common/static/common/templates/discussion/thread-list-item.underscore

```diff
@@ -72,7 +72,18 @@
       %>
       </span>
 
-      <span class="forum-nav-thread-comments-count <% if (unread_comments_count > 0) { %>is-unread<% } %>">
+      <% if (!neverRead && unread_comments_count > 0) { %>
+        <span class="forum-nav-thread-unread-comments-count">
+          <%-
+            StringUtils.interpolate(
+              gettext('{unread_comments_count} new'),
+              {unread_comments_count: unread_comments_count}
```

Comment:
That's how I used to write my code, but our edx lint rules override the object-curly-spacing rule to say that you can't have spaces. I've recently been teaching myself to do it without the spaces!

https://www.npmjs.com/package/eslint-config-edx

I don't remember why we decided to override the AirBnB rule which requires spaces like you say.